demisto_sdk/commands/validate/tests/BC_validators_test.py

```diff
@@ -891,3 +897,138 @@ def test_IsContextPathChangedValidator_remove_command():
+    )
+    assert (
+        "contain changes to the names of existing arguments: In command 'test-command_2' the following arguments have been changed: old_arg_2. Please undo the changes."
+        in results[1].message
+    )
+
+
+def test_HaveCommandsOrArgsNameChangedValidator__passes():
+    """
+    Given
+    - A new content item with 3 commands. all commands have only 1 argument except the third command which has 2 arguments.
+    - An old content item with the same structure as the new content item, but with addition of a new command an new argument to existing commands.
+    When
+    - Calling the HaveCommandsOrArgsNameChangedValidator.
+    Then
+    - Make sure the validation passes and no error messages are returned, since adding new commands or arguments is allowed.
+    """
+    # Setup new content item with changes in command and argument names
+    new_content_item = setup_integration_with_3_commands_and_5_args()
+
+    new_content_item.commands.append(create_integration_object().commands[0])
+    new_content_item.commands[2].args.append(
+        create_integration_object().commands[0].args[0]
+    )
+    new_content_item.commands[3].name = "new_command_3"
+    new_content_item.commands[2].args[2].name = "new_arg_2"
+
+    # Setup old content item with original command and argument names
+    old_content_item = setup_integration_with_3_commands_and_5_args()
+
+    create_old_file_pointers([new_content_item], [old_content_item])
+    assert not HaveCommandsOrArgsNameChangedValidator().is_valid([new_content_item])
+
+
+def test_NewRequiredArgumentValidator__fails():
+    """
+    Given
+    - A old content item with 3 commands. all commands have only 1 argument except the third command which has 2 arguments.
+    - A new content item with the same structure as the old content item, but with changes in the second command argument to be required and a new required argument in the third command.
+    When
+    - Calling the NewRequiredArgumentValidator.
+    Then
+    - Make sure the validation fails and the right error message is returned notifying on the new required argument in the second command and the third command.
+    """
+    new_content_item = setup_integration_with_3_commands_and_5_args()
+    new_content_item.commands[2].args.append(
+        create_integration_object().commands[0].args[0]
+    )
+    # change existing arg to be required
+    new_content_item.commands[1].args[0].required = True
+    # add new required arg
+    new_content_item.commands[2].args[2].required = True
+    old_content_item = setup_integration_with_3_commands_and_5_args()
+
+    create_old_file_pointers([new_content_item], [old_content_item])
+    res = NewRequiredArgumentValidator().is_valid([new_content_item])
+    assert (
+        "in command 'test-command_1' you have added a new required argument: 'from_date'.in command 'test-command_2' you have added a new required argument: 'from_date'. Please undo the changes."
```

Comment:
```suggestion
        "in command 'test-command_1' you have added a new required argument: 'from_date'.in command 'test-command_2' you have added a new required argument: 'from_date'. Please undo the changes."
```
''from_date'.in command '
Make sure to alter the validation code to create space between the '.' and 'in'
I think splitting by \n would be more readable in that case.

Reply:
done